Dockerfile

```diff
@@ -62,6 +72,11 @@ RUN apt-get install -y \
       x11proto-print-dev \
       xutils-dev
 
+# Install libpng12
+RUN wget -q -O /tmp/libpng12.deb http://mirrors.kernel.org/ubuntu/pool/main/libp/libpng/libpng12-0_1.2.54-1ubuntu1_amd64.deb && \
```

Comment:
Equivalent to `curl --silent --output /tmp/libpng12.deb http://mirrors.kernel.org/ubuntu/pool/main/libp/libpng/libpng12-0_1.2.54-1ubuntu1_amd64.deb`, I think, which means wget isn't needed.

Reply:
Resolved in b5b1714 + 5bd660d